pkg/verifier/aws/aws_verifier.go

```diff
@@ -61,8 +61,17 @@ type AwsVerifier struct {
 	Output    output.Output
 }
 
-func GetAMIForRegion(region string) string {
```

Comment:
Note from call today: OCM is using this and can't be removed.

Reply:
Added back, with a comment explaining its purpose so it won't get deleted in the future